src/provisioner/cluster.py

```diff
@@ -16,6 +16,8 @@
 import errno
 from cortx.provisioner.error import CortxProvisionerError
 from cortx.utils.validator.error import VError
+from cortx.provisioner import PLog
```

Comment:
We need the endpoint to be Log only. Log can either be printed onto the screen or goto a log file. Interface should be same, i.e. Log. 

There will be a Log.Init() call to direct to file or screen.

Reply:
Reverted. 
Done.